src/scripts/seed-db-test-decoded-txs.ts

```diff
@@ -0,0 +1,39 @@
+import { filterABIMap } from 'utils'
+import { MongooseDatabaseInterface } from 'utils/mongoose'
+import txHashArr from '../__tests__/testTxHashArr.json'
+import getTranslator from '../utils/translator';
+import { DecodedTx } from 'interfaces/decoded';
+
+async function run() {
+  const db = new MongooseDatabaseInterface('mongodb://localhost:27017/evm-translator')
+  await db.connect()
+
+  const translator = await getTranslator()
+
+  const decodedTxArr: DecodedTx[] = [];
+  for (const txHash of txHashArr) {
+    const rawTxData = await translator.getRawTxData(txHash)
```

Comment:
I think there's a single function on `translator` that does all of these that we can call here instead